abis/mlibc/inotify.h

```diff
@@ -0,0 +1,7 @@
+#ifndef _ABIBITS_INOTIFY_H
+#define _ABIBITS_INOTIFY_H
+
+#define __MLIBC_IN_CLOEXEC 1
+#define __MLIBC_IN_NONBLOCK 2
```

Comment:
Let's not add any more `__MLIBC` ABI constants, the `__MLIBC_O_PATH` etc. ones in `abi-bits/abi.h` were a mistake (by me :smile:).